sys/net/gnrc/network_layer/ipv6/nib/_nib-6ln.c

```diff
@@ -281,13 +281,10 @@ uint32_t _handle_6co(const icmpv6_hdr_t *icmpv6,
     uint8_t cid;
 #endif  /* MODULE_GNRC_SIXLOWPAN_CTX */
 
-    if ((sixco->len != SIXLOWPAN_ND_OPT_6CTX_LEN_MIN) ||
-        ((sixco->len != SIXLOWPAN_ND_OPT_6CTX_LEN_MAX) &&
-         (sixco->ctx_len > 64U)) ||
-        (icmpv6->type != ICMPV6_RTR_ADV)) {
-        DEBUG("nib: received 6CO of invalid length (%u), must be %u "
-              "or wasn't delivered by RA."
-              "\n",
+    assert(icmpv6->type == ICMPV6_RTR_ADV);
```

Comment:
That's from a received package, right?
Since there is always the possibility to receive garbage, we can't trigger an assert on a malformed packet, otherwise a bad node can bring other, perfectly fine behaving nodes down by sending packets that trigger asserts.

Reply:
That´s why I found the check unnecessary.

Reply:
So if the assert ever fires, we would have a misuse of the function rather than an evil node sending us garbage :)